frontend/src/pages/create_campaign/Campaign/index.js

```diff
@@ -0,0 +1,161 @@
+import React from "react";
+import { FormControlLabel, Switch } from "@mui/material";
+import PropTypes from "prop-types";
+import AdapterDateFns from "@mui/lab/AdapterDateFns";
+import LocalizationProvider from "@mui/lab/LocalizationProvider";
+import DateTimePicker from "@mui/lab/DateTimePicker";
+import Dropzone from "react-dropzone";
+import {
+  CampaignContainer,
+  CampaignDropzone,
+  CampaignRowDiv,
+  SwitchRowDiv,
+  CampaignTextField,
+} from "./createCampaign.styled";
+import { fileToDataUrl } from "../../../utils";
+
+const CampaignTab = (props) => {
```

Comment:
i'd prefer
```
const CampaignTab = ({
    index,
    value,
    campaignName,
    setCampaignName,
    startDate,
    setStartDate,
    endDate,
    setEndDate,
    description,
    setDescription,
    interviewStage,
    setInterviewStage,
    scoringStage,
    setScoringStage,
    cover,
    setCover,
  }) => {
  ```
  
for better readability

Reply:
I agree that this is better tbh, though I'm pretty sure most of the existing code uses "props" as an arg and then destructures in the function body D: